docker-compose.azure.yml

```diff
@@ -1,10 +1,9 @@
-version: '3.6'
+version: "3.6"
```

Comment:
This is [prettier](https://prettier.io)'s fault; I've had to manually tell it not to reformat the entirety of `azure-pipelines.yml` and `azure/template.json`, but I think we should let it autoformat those too sometime.